pkg/utils/test/config.go

```diff
@@ -42,6 +42,7 @@ func DefaultControllerConfigForTests(opts ...ControllerConfigOption) manager.Con
 	cfg.ServiceAccountToImpersonate = ServiceAccountToImpersonate
 	// TODO: https://github.com/Kong/kong-operator/issues/1986
 	cfg.ConversionWebhookEnabled = false
+	cfg.ValidationWebhookEnabled = false
```

Comment:
Can we have a similar issue for validation webhook like we have https://github.com/Kong/kong-operator/issues/1986 for conversion?

Reply:
https://github.com/Kong/kong-operator/issues/2029